### ISSUE TITLE
Test failing but still the build runs green

### ISSUE THREAD

**Issue by theBadT** [theia-ide/theia#4683]
### Description
As you can see from https://ci.appveyor.com/project/kittaakos/theia/builds/23307786#L1157
test is failing, but still the build runs green. The same issue occurs locally. Meaning, test fails -> overall result is green.
This causes uncertanity regarding the test execution. Should it fail intentionally?

### Reproduction Steps
yarn run test, with theia/core being the root folder

**OS and Theia version:** Windows 10 Pro, latest master

**Diagnostics:**

**Comment by kittaakos** [comment]
> As you can see from

Where is the test failure?

**Comment by theBadT** [comment]
Request fails failed with error: fails failed Error: fails failed\n    at TestServer.fails (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.spec.js:57:15)\n    at JsonRpcProxyFactory.<anonymous> (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:217:73)\n    at step (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:47:23)\n    at Object.next (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:28:53)\n    at C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:22:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:18:12)\n    at JsonRpcProxyFactory.onRequest (C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:211:16)\n    at C:\projects\theia\packages\core\lib\common\messaging\proxy-factory.js:174:48\n    at handleRequest (C:\projects\theia\node_modules\vscode-jsonrpc\lib\main.js:357:42)\n    at processMessageQueue (C:\projects\theia\node_modules\vscode-jsonrpc\lib\main.js:252:17)\n    at Immediate.<anonymous> (C:\projects\theia\node_modules\vscode-jsonrpc\lib\main.js:242:13)\n    at runCallback (timers.js:810:20)\n    at tryOnImmediate (timers.js:768:5)\n    at processImmediate [as _immediateCallback] (timers.js:745:5)

so basically, since there is no I'm assuming this test has failed
